packages/@amazeelabs/silverback-search/turbo.json

```diff
@@ -2,7 +2,40 @@
   "$schema": "https://turborepo.org/schema.json",
   "extends": ["//"],
   "tasks": {
+    "test:static": {
```

Comment:
All Drupal-related `turbo.json` files look exactly the same. Could we create a central one and just use an empty one that just uses "extends" within the packages?